### PR TITLE
Add chapter 9 notes

### PR DESCRIPTION
**Pull request by ZhugeSong**

### CODE REVIEW

File: zim-wiki/Chapters.txt [hunk 1]
```diff
@@ -236,7 +236,7 @@ Creation-Date: 2017-05-10T22:37:59-07:00
 * Bruic appears and Dimitri charismas her
 * Luna tells Bruic that the haunted town is now cleared, and she departed
 * Ashling returns and announces she is coming with the party
-* Dimitri privately asks Ashling to help him find [[Characters:Ariel|Ariel]]
```
Comment (firedraco) [inline zim-wiki/Chapters.txt:239]: Should be part of a separate commit

**Review by firedraco — CHANGES_REQUESTED** [review]

File: zim-wiki/Chapters.txt (hunk 1 above)
Comment (ZhugeSong) [reply zim-wiki/Chapters.txt:239]: Yup
